### ISSUE TITLE
Check Home Assistant configuration

### ISSUE THREAD

**Issue by skaker1** [isabellaalstrom/sensor.krisinformation#22]
Hi!

Checking for Home Assistant 0.106.6:

Failed config
General Errors:
- Platform error sensor.krisinformation - No module named 'xmltodict'

**Comment by isabellaalstrom** [comment]
Have you upgraded and seen if there is actually any problems? I can't get to this right now.

**Comment by skaker1** [comment]
I tried, but it started in safe mode and I couldn't get it to work. I
reversed to a snapshot.

Best regards

Kenneth Eriksson


Den lör 14 mars 2020 kl 10:08 skrev Isabella Gross Alström <
notifications@github.com>:

**Comment by fredrikbaberg** [comment]
I'm seeing the same issue (I think).
On running the add-on "Check Home Assistant configuration", it fails with:
```
Failed config
  General Errors: 
    - Platform error sensor.krisinformation - No module named 'xmltodict'
```

This does not break Home Assistant for me, I've been able to update, but I'm getting failures when running the addon to check config. I can't see xmltodict being used in the code though?

Krisinformation installed through HACS, version: 4390969
Home Assistant 0.107.6

**Comment by isabellaalstrom** [comment]
Fixed in latest release 1.0.1.
